eventhub/client.go

```diff
@@ -103,19 +105,27 @@ func SubscribePartitions(ctx context.Context, sess *amqp.Session, name, group st
 // PutTokenContinuously writes token first time in blocking mode and returns
 // maintaining token updates in the background until stopCh is closed.
 func (c *Client) PutTokenContinuously(
-	ctx context.Context,
 	audience string,
 	token string,
+	cred *common.Credentials,
 	stopCh chan struct{},
 ) error {
-	if err := c.PutToken(ctx, audience, token); err != nil {
+	if err := c.PutToken(context.Background(), audience, token); err != nil {
```

Comment:
I think you shouldn't have dropped ctx for the first `c.PutToken` invocation because it's viable part of the connection process.

Reply:
I think so too! fixed 💪 https://github.com/goautomotive/iothub/pull/2/commits/43de6487238b8c36411fdf76f851146b9516ad0d